webapp/tests/karma/ts/components/search-bar/search-bar.component.spec.ts

```diff
@@ -9,6 +9,7 @@ import { FreetextFilterComponent } from '@mm-components/filters/freetext-filter/
 import { Selectors } from '@mm-selectors/index';
 import { ResponsiveService } from '@mm-services/responsive.service';
 import { SearchFiltersService } from '@mm-services/search-filters.service';
+import { FormsModule } from '@angular/forms';
```

Comment:
Tiny, but It'd be super nice to keep the imports organized by moving this to line 2. The first block of imports is for third parties/libs, second block is for this project files.